tools/kubernetes/docker/shuffle-file-cleaner/build-cleaner-image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#TODO hard code for now
+cd "${dirname "$0"}"
+cp -r ../../../../jars/* ./jars/
```

Comment:
??? what's this?

Reply:
Try to fetch the dist jars for ${KYUUBI_HOME}/jars, but this shell local ${KYUUBI_HOME}/tools/kubernetes/docker/shuffle-cleaner.